src/main/java/org/threadly/litesockets/NoThreadSocketExecuter.java

```diff
@@ -337,6 +340,7 @@ private void doRead(SelectableChannel sc) {
           removeClient(client);
           client.close();
         } else if( read > 0) {
+          stats.addWrite(read);
```

Comment:
This looks like it should be addRead instead of addWrite.  Might be a good spot to write a unit test against.